flink-table-store-spark/src/main/java/org/apache/flink/table/store/spark/SparkScan.java

```diff
@@ -128,13 +124,13 @@ public boolean equals(Object o) {
         }
 
         SparkScan that = (SparkScan) o;
-        return table.location().equals(that.table.location())
+        return table.name().equals(that.table.name())
```

Comment:
Is it safe to just compare table's name? What about two tables with the same name and schema but under different namespace?

Reply:
In one catalog, different tables should have different names.